frontend/public/index.html

```diff
@@ -60,5 +60,20 @@
     <noscript>JavaScript must be enabled.</noscript>
     <div id="popper-container"></div>
     <div id="app"></div>
+    <script>
+      const theme = localStorage.getItem('bridge/theme');
+      let systemTheme;
+      if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        systemTheme = 'dark';
+      }
+      if (
+        (theme === 'dark') ||
+        (theme === 'systemDefault' && systemTheme === 'dark')
+      ) {
+        htmlTagElement.classList.add('pf-theme-dark');
+      } else {
+        htmlTagElement.classList.remove('pf-theme-dark');
```

Comment:
Where comes `htmlTagElement` from ???

Reply:
Cypress noticed this as well ;)

![image](https://user-images.githubusercontent.com/139310/157687097-f8be3532-b39e-4fe2-8cf7-6dbc405afeb4.png)

Reply:
Thanks, Updated the code. Not sure how it gets deleted.